src/routes/charities/[program_code]/+page.svelte

```diff
@@ -4,8 +4,194 @@
 	export let data: PageData;
 
 	$: program = data.charityProgram;
+
+	let value: number | null = null;
+
+	const handleAmountTokenClick = (amount: number) => {
+		value = amount;
+	};
+
+	const amountTokens = [
+		{
+			title: '50 ₽',
+			amount: 50,
+		},
+		{
+			title: '100 ₽',
+			amount: 100,
+		},
+		{
+			title: '200 ₽',
+			amount: 200,
+		},
+		{
+			title: '500 ₽',
+			amount: 500,
+		},
+	];
 </script>
 
-<p>Organization: {program.fundName}</p>
-<p>Name: {program.serviceName}</p>
-<p>Description: {program.serviceDescription}</p>
+<div class="charityPay">
+	<div class="charityPay__content">
+		<span class="charityPay__subtitle">{program.fundName}</span>
+		<div class="charityPay__header">
+			<h1 class="charityPay__title">{program.serviceName}</h1>
+			<a href="/charities" class="charityPay__link">Back</a>
+		</div>
+		<p class="charityPay__description">{program.serviceDescription}</p>
+		<div class="charityPay__pay">
+			<h2 class="charityPay__pay_title">Select amount to donate</h2>
+			<input placeholder="250 ₽" class="charityPay__pay_input" type="number" {value} />
+			<div class="charityPay__pay_amounts">
+				{#each amountTokens as { title, amount }}
+					<!-- svelte-ignore a11y-click-events-have-key-events -->
+					<span on:click={() => handleAmountTokenClick(amount)} class="charityPay__pay_token">
```

Comment:
This function is too simple, we should probably just inline it:
```suggestion
					<span on:click={() => value = amount} class="charityPay__pay_token">
```